src/ml/neural_net/compute_context.hpp

```diff
@@ -126,9 +126,8 @@ EXPORT class compute_context {
    * \todo what args here?
    */
   virtual std::unique_ptr<model_backend> create_drawing_classifier(
-      /* TODO: const float_array_map& weights, const float_array_map& config
-       * Until the nn_spec in C++ isn't ready, do not pass in any weights.
-       */
+      /* TODO: const float_array_map& config if needed */
```

Comment:
For future reference, in hindsight it would have been better to pass empty weights upfront and ignore them in the implementation, then have to change the interface everywhere.

Reply:
Noted, thanks.